README.md

```diff
@@ -21,19 +21,18 @@ trivial to integrate with web services.
 
 ```php
 $client = new \GuzzleHttp\Client();
-$res = $client->request('GET', 'https://api.github.com/repos/guzzle/guzzle');
-echo $res->getStatusCode();
-// 200
-echo $res->getHeaderLine('content-type');
-// 'application/json; charset=utf8'
-echo $res->getBody();
-// '{"id": 1420053, "name": "guzzle", ...}'
-
-// Send an asynchronous request.
+$request = $client->request('GET', 'https://api.github.com/repos/guzzle/guzzle');
+
+echo $request->getStatusCode(); # 200
+echo $request->getHeaderLine('content-type'); # 'application/json; charset=utf8'
+echo $request->getBody(); # '{"id": 1420053, "name": "guzzle", ...}'
```

Comment:
this does not make sense.. it's not a request.. but a response

Reply:
Ok, I create pr #2251